src/components/quiz/questions.ts

```diff
@@ -0,0 +1,93 @@
+export interface Question {
+  id: number;
+  Question: string;
+  Answers: string[];
+  Answer: number;
```

Comment:
Prefer lowercase for object keys
```suggestion
  question: string;
  answers: string[];
  answer: number;
```